docs/flowdiagram.svg

```diff
@@ -0,0 +1,3 @@
3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 56px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;"><span><font style="font-size: 15px">Lambda<br /></font></span></div></div></foreignObject><text x="27" y="14" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><g transform="translate(180.5,489.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="71" height="26" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 71px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;">registers service</div></div></foreignObject><text x="36" y="19" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica">registers service</text></switch></g><g transform="translate(322.5,374.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="71" height="26" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 71px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;">subscribes events</div></div></foreignObject><text x="36" y="19" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica">subscribes events</text></switch></g><ellipse cx="50.5" cy="563" rx="15" ry="15" fill="#ffffff" stroke="#24cacc" pointer-events="none"/><g transform="translate(76.5,556.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="123" height="12" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 123px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;">Connectors we provide</div></div></foreignObject><text x="62" y="12" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica">Connectors we provide</text></switch></g></g></svg>
```

Comment:
Change the arrow with "sends events" label - it should go from github to github connector, not the other way.